### PR TITLE
Mari definition update

### LINKED ISSUE #17943: MARI instrument definition changes
MARI has new vanadium standard samples with slightly different mass to the old ones. 

The values of the mass in the parameter files should be changed so that absolute normalisation works with the new standard samples.

### PR DESCRIPTION
**Pull request by mducle**
**Description of work.**

MARI has been upgraded with a new guide in 2017. This changes the instrument definition and parameters files to reflect the new instrument. The main changes are the position of the monitors used to calculate the energy.

The vanadium sample mass has changed but this is not very significant. Measurements indicate that the value in the parameter file can remain the same.

**Report to:** [user name]/[nobody]. 

**To test:**

Fixes #17943. 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Review by OwenArnold — DISMISSED** [review]
Changes look reasonable. Builds passing. Approved.

File: instrument/MARI_Definition.xml [hunk 1]
```diff
@@ -4,9 +4,9 @@
 <instrument xmlns="http://www.mantidproject.org/IDF/1.0" 
             xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
             xsi:schemaLocation="http://www.mantidproject.org/IDF/1.0 http://schema.mantidproject.org/IDF/1.0/IDFSchema.xsd"
- name="MARI" valid-from   ="1900-01-31 23:59:59"
-                        valid-to     ="2100-01-31 23:59:59"
-                        last-modified="2015-03-01 00:00:00">
+ name="MARI" valid-from   ="2016-09-11 23:59:59"
+                        valid-to     ="2100-01-01 23:59:59"
+                        last-modified="2016-11-02 15:50:00">
```
Comment (gvardany) [inline instrument/MARI_Definition.xml]: Is this not supposed to be the date of last modification of the file?
Also I think `valid-to` is optional here.

File: instrument/MARI_Definition_19900101_20160911.xml [hunk 2]
```diff
@@ -0,0 +1,1405 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- For help on the notation used to specify an Instrument Definition File 
+     see http://www.mantidproject.org/IDF -->
+<instrument xmlns="http://www.mantidproject.org/IDF/1.0" 
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xsi:schemaLocation="http://www.mantidproject.org/IDF/1.0 http://schema.mantidproject.org/IDF/1.0/IDFSchema.xsd"
+ name="MARI" valid-from   ="1900-01-31 23:59:59"
```
Comment (gvardany) [inline instrument/MARI_Definition_19900101_20160911.xml]: And `valid-from` must be optional here.

File: instrument/MARI_Parameters_19900101_20160911.xml [hunk 3]
```diff
@@ -0,0 +1,480 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<parameter-file instrument = "MARI" valid-from = "2013-06-12T00:00:00">
```
Comment (gvardany) [inline instrument/MARI_Parameters_19900101_20160911.xml]: Also I think it read nicer like this rather than 23:59:59 of the previous day.

**Review by gvardany — CHANGES_REQUESTED** [review]
Is the IDF generated by a script in the `mantidgeometry`? If so, has it also been updated correspondingly?

**Comment by mducle** [comment]
> Is the IDF generated by a script in the `mantidgeometry`? If so, has it also been updated correspondingly?

I don't know how the IDF was originally generated... but it hasn't been updated. 

I would really like the changes to the monitor positions put into Mantid ASAP though because it affects the reduction and we have users coming this week.

**Comment by mducle** [comment]
IDFs seems to need both date from and date to fields else they fail the `APITest_ExperimentInfoTest`

**Comment by gvardany** [comment]
Hmm, this does not match with the documentation, but anyhow, seems I was wrong that they are optional.

**Review by gvardany — APPROVED** [review]

**Pull request merged**